internal/config/gofer/gofer.go

```diff
@@ -62,6 +62,7 @@ func (e ErrCyclicReference) Error() string {
 
 type Gofer struct {
 	RPC         RPC                   `json:"rpc"`
+	EthRPC      string                `json:"ethRpc"`
```

Comment:
Could we use just `internal/config/ethereum` instead to load the RPC address?

Reply:
No. These are different concepts in the context of the whole suite.
`internal/config/ethereum` is for interacting with the chain the system is worknig on (so could be mainnet or testnets) and the `ethRpc` is always for mainnet so we can get real data into testing environments.

I see some room for improving these concepts when we move to multi-chain operations.